### PR TITLE
add metaflow-stubs

### PR DESCRIPTION
**Pull request by madhur-ob**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by saikonen — APPROVED** [review]
looks good to me, straightforward recipe.

File: recipes/metaflow-stubs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - metaflow_stubs
```
Comment (saikonen) [inline recipes/metaflow-stubs/meta.yaml:26]: This test is failing due to the package not being importable.

```
ModuleNotFoundError: No module named 'metaflow_stubs'
```

File: recipes/metaflow-stubs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  commands:
```
[the remark this answers is not in the document]
Comment (madhur-ob) [reply recipes/metaflow-stubs/meta.yaml:25]: not sure...
also, not all files are tested, only `__init__.py` along with all the directories...
the idea is basically for at least SOME files and directories to exist in the right place...

We cannot maintain all files and it is not the intention too...

[the remark this answers is not in the document]
Comment (savingoyal) [reply recipes/metaflow-stubs/meta.yaml:25]: +1 to sakari - these files may move around at any point breaking the conda build

[the remark this answers is not in the document]
Comment (madhur-ob) [reply recipes/metaflow-stubs/meta.yaml:25]: We can just test for a subset of them? that will always exist?
The pattern with such recipes is always to check for a subset of files, and not all of them.

**Review by savingoyal — APPROVED** [review]
lgtm!

**Pull request merged**